Sming/Components/Network/src/Network/Http/HttpRequestAuth.cpp

```diff
@@ -12,7 +12,7 @@
 
 #include "HttpRequestAuth.h"
 #include "HttpRequest.h"
-#include "Network/WebHelpers/base64.h"
+#include <Data/WebHelpers/base64.h>
```

Comment:
@mikee47 Please start an upgrade document for this release and add information that `Network/WebHelpers/base64.h` is now moved to `Data/WebHelpers/base64.h`.

Reply:
You happy with this change? We could leave the path as-is... or provide an alias so it doesn't break anything...

Reply:
> or provide an alias so it doesn't break anything

I am ok with the change. It would be great if you can add the old header file pointing to the new one with a deprecation notice for smoother transition. Similar to what we have here: https://github.com/SmingHub/Sming/blob/d375ef74fd11f0796aa4483c0bdaacb4fb755655/Sming/SmingCore/SmingCore.h